notebooks/official/feature_store/feature_store_streaming_ingestion_sdk.ipynb

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        "body_mass_g": {"value_type": "DOUBLE"},
</pre><p>format body_mass_g and sex like the above features</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>

Reply:
<p>Good catch! Done.</p>